src/select/searchable.rs

```diff
@@ -603,29 +598,59 @@ where
                                                 </td>
                                             </tr>
                                         }
-                                    } else if item.value_txt(&txt, ctx.props().language) == value {
-                                        html!{
-                                            <tr class="searchable-select-list-item-single" onclick={onclick_item(item.id().clone())}>
-                                                <td class="searchable-select-list-item-single-selected">
-                                                    { item_value }
-                                                </td>
-                                            </tr>
+                                    })
+                                }
+                            } else {
+                                html! {
+                                    for list.iter().map(|item| {
+                                        let check_status = if let Ok(selected) = ctx.props().selected.try_borrow() {
+                                            selected.as_ref().map_or(CheckStatus::Checked, |selected|
+                                                if selected.contains(item.id()) {
+                                                    CheckStatus::Checked
+                                                } else {
+                                                CheckStatus::Unchecked
+                                            })
+                                        } else {
+                                            CheckStatus::Unchecked
+                                        };
```

Comment:
This part doesn't look aligned in the best way.

Reply:
This part has not changed from the previous code except for spaces(indentation). Therefore, I have modified the spaces to be the same as the previous code.